python/ray/serve/autoscaling_policy.py

```diff
@@ -130,7 +130,7 @@ def replica_queue_length_autoscaling_policy(
 
         # Only actually scale the replicas if we've made this decision for
         # 'scale_up_consecutive_periods' in a row.
-        if decision_counter > int(config.upscale_delay_s / CONTROL_LOOP_PERIOD_S):
+        if decision_counter > int(config.upscale_delay_s / CONTROL_LOOP_INTERVAL_S):
```

Comment:
Seems like the interval is used in a few other places to count control loop cycles - am I breaking some assumption by allowing it to be configurable to some larger value (e.g., does this still make sense if the loop interval is large)?

Reply:
I don't believe so -- but @zcin should confirm